### DISCUSSION TITLE
If I want to Klipperized my SV06 Plus with BTT Pi V1.2 + TFT35 SPI: Same procedere with "One-Stop-Shop Klipper Configuration" as with BTT Pad 7?

### DISCUSSION THREAD

**Discussion by cyaneo** [bassamanator/Sovol-SV06-firmware#156] [General]
Hello,
I want to Klipperized my SV06 Plus.

Can I use the same procedure as on BTT Pad 7 when using BTT Pi V1.2 + TFT35 SPI?

Thank you!

**Comment by bassamanator** [comment]
I'm sure the process will be very similar. I don't own either of those devices so I cannot say more.

**Comment by cyaneo** [comment]
OK, Thank you, the process is very similar, but I can't print yet ("Unable to open file") - stucking on gcode path problem (moonraker.log):

```
Received: '/home/pi/printer_data/gcodes'
Expected: '/home/biqu/printer_data/gcodes'
Modify the [virtual_sdcard] section Klipper's configuration to correct this error.
```

in printer.cfg I added
[virtual_sdcard]
path: /home/biqu/printer_data/gcodes

but it didn't solve the problem.

Any further tipps?

**Comment by bassamanator** [comment]
What is the new error reported? And have you confirmed that that path exists?

**Reply by cyaneo** [reply]
the gcodes path is "/home/biqu/printer_data/gcodes" (there are the gcodes) but moonraker Received "/home/pi/printer_data/gcodes"

but I can't find out where the path "/home/pi/printer_data/gcodes" comes from (and this path does not exist)

**Comment by bassamanator** [comment]
You didn't answer either of my questions.

You need to read the before you begin section.

**Reply by cyaneo** [reply]
- The path "/home/pi/printer_data/gcodes" does not exist
- The Error reports a wrong path in moonraker.log

**Comment by bassamanator** [comment]
It's still reporting
```
Received: '/home/pi/printer_data/gcodes'
Expected: '/home/biqu/printer_data/gcodes'
```
You need to read the 9 minutes worth of documentation to avoid such issues.

**Reply by cyaneo** [reply]
Bassam, I got it! Now it works, thank you.

I just dismiss
"Robert Redford's performance in Spy Game (2001) was superb!" (or RTFM).
